Azure-Samples/sample-mdw-serverless
# parameters to sql + new pipeline

## Purpose
added the pipeline using the function
updated sql scripts to reflect parameters

## Does this introduce a breaking change?

```
[ ] Yes
[X ] No
```

## Pull Request Type
New pipeline that will move data from bronze2sliver using an azure function, that will also divide the content to the right date partition.

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
clone
run the bicep scripts to init an env
connect the workspace to your cloned repo
update the name of the repo in the github action (autoinject_params)
publish the branch
test in the newly created workspace
* Test the code
verify your azure function is up and running
start the new pipeline
```
```

## What to Check
Verify that the following are valid
* files are moved to silver, dates in the file match the date partition
